ProteoformSuiteInternal/Proteoform.cs

```diff
@@ -94,7 +94,8 @@ public List<ExperimentalProteoform> identify_connected_experimentals(List<PtmSet
                         null); //Experimental without theoretical reference
                 string theoretical_base_sequence = theoretical_base != null ? theoretical_base.sequence : "";
 
-                PtmSet best_addition = generate_possible_added_ptmsets(r.peak.possiblePeakAssignments, deltaM, mass_tolerance, all_mods_with_mass, theoretical_base, theoretical_base_sequence, 1)
+                List<PtmSet> possible_additions = r.peak.possiblePeakAssignments.Where(p => Math.Sign(p.mass) == Math.Sign(deltaM)).ToList();
```

Comment:
I think the only PTM list close to 0 worth considering is deamidation right? And this should only be negative?